content/200-concepts/200-database-connectors/08-sql-server/index.mdx

```diff
@@ -1,49 +1,19 @@
 ---
-title: 'Microsoft SQL Server connector'
-metaTitle: 'Microsoft SQL Server Connector (Preview)'
-metaDescription: 'Preview content for the Microsoft SQL Server Connector.'
+title: 'Microsoft SQL Server'
+metaTitle: 'Microsoft SQL Server'
+metaDescription: 'This page explains how Prisma can connect to a Microsoft SQL Server database using the Microsoft SQL Server database connector.'
 ---
 
 <TopBlock>
 
-The Microsoft SQL Connector is available as a Preview feature in [2.10.0](https://github.com/prisma/prisma/releases/tag/2.10.0).
+The Microsoft SQL Server data source connector connects Prisma to a [Microsoft SQL Server](https://docs.microsoft.com/en-us/sql/sql-server/?view=sql-server-ver15) database server.
 
 </TopBlock>
 
-## Enable SQL Server Preview feature
-
-To enable the Microsoft SQL Server Preview feature:
-
-1. Update the `client` and `datasource` blocks in your schema as shown:
-
-   ```prisma file=prisma/schema.prisma highlight=3,7;normal
-   generator client {
-     provider        = "prisma-client-js"
-     previewFeatures = ["microsoftSqlServer"]
-   }
-   
-   datasource db {
-     provider = "sqlserver"
-     url      = env("DATABASE_URL")
-   }
-   ```
-
-1. Generate Prisma Client:
-
-   ```terminal
-   npx prisma generate
-   ```
-
-Alternatively, follow the [Microsoft SQL Server tutorial](sql-server-start-from-scratch-typescript).
-
 ## Type mapping between Microsoft SQL Server to Prisma schema
 
 For type mappings organized by Prisma type, refer to the [Prisma schema reference](../../../../reference/api-reference/prisma-schema-reference#model-field-scalar-types) documentation.
 
-## Supported versions
-
-See [Supported databases](../../../../reference/database-reference/supported-databases#preview).
-
```

Comment:
Why did this go away? Supported versions of SQL Server are still relevant?

Reply:
I removed it as my thinking was its linking to the preview. But i changed that page to move sql server from preview into supported and forgot to put the link back